internal/v1/handler_recommend_package.go

```diff
@@ -29,6 +29,10 @@ func (h *Handlers) handleRecommendationsResponse(ctx echo.Context, uploadPackage
 	cloudRP := recommendations.RecommendPackageRequest{
 		Packages:            uploadPackageRequest.Packages,
 		RecommendedPackages: uploadPackageRequest.RecommendedPackages,
+		Distribution: recommendations.DistributionItem{
+			Name:        uploadPackageRequest.Distribution.Name,
+			Description: uploadPackageRequest.Distribution.Description,
+		},
```

Comment:
Could you also rename `uploadPackageRequest` to something more appropriate. If possible shorter.